metadata_backend/conf/conf.py

```diff
@@ -207,7 +226,7 @@ def create_db_client() -> AsyncIOMotorClient:
 DATACITE_SCHEMAS = {"study", "dataset", "bpdataset"}
 METAX_SCHEMAS = {"study", "dataset"}
 
-mq_ssl = bool(strtobool(os.environ.get("BROKER_SSL", "False")))
+mq_ssl = bool(str_to_bool(os.environ.get("BROKER_SSL", "False")))
```

Comment:
```suggestion
mq_ssl = str_to_bool(os.environ.get("BROKER_SSL", "False"))
```
reduntant `bool()` (already boolean)